pkg/bugzilla/types.go

```diff
@@ -56,15 +57,16 @@ func (s *Score) UnmarshalJSON(b []byte) error {
 // Bug maps to the desired fields of a bugzilla bug
 type Bug struct {
 	Id            int             `json:"id"`
-	Status        string          `json:"status"`
-	Summary       string          `json:"summary"`
 	Component     SingleElemSlice `json:"component"`
 	TargetRelease SingleElemSlice `json:"target_release"`
 	AssignedTo    string          `json:"assigned_to"`
-	Keywords      []string        `json:"keywords"`
+	Status        string          `json:"status"`
+	Keywords      pq.StringArray  `json:"keywords"`
 	PmScore       Score           `json:"cf_pm_score"`
+	Summary       string          `json:"summary"`
 	Externals     json.RawMessage `json:"external_bugs"`
 	DateStamp     time.Time
+	Age           int
```

Comment:
Implement in a separate PR.  Is going to require database migration, an updated table, and several changes to the db and api packages.

Reply:
Resolved. No DB migration necessary, yay for views.